### ISSUE TITLE
breaking line after parenthesis

### ISSUE THREAD

**Issue by jjangga0214** [darold/pgFormatter#134]
Hi, thank you for this convenient formatter :)

I found unexpected behavior.
While I want the formatting like this,

```sql
CREATE TABLE Foo (
    created_at timestamp WITH time zone DEFAULT now() NOT NULL,
    updated_at timestamp WITH time zone DEFAULT now() NOT NULL
);
```

**pgFormatter** actually formats like the below.

```sql
CREATE TABLE Foo (
    created_at timestamp WITH time zone DEFAULT now(
) NOT NULL,
    updated_at timestamp WITH time zone DEFAULT now(
) NOT NULL
);
```

I think it would be cool to fix this. How do you think? 

Thank you!

**Comment by darold** [comment]
Commit b98b77a might just fix it.

**Comment by jjangga0214** [comment]
Great!
I hope this fix will be applied to tools that use pgFormatter, such as [vscode-pgFormatter](https://github.com/bradymholt/vscode-pgFormatter). Could I ask when would be the next release(e.g. 4.0.1 or 4.1.0)?

**Comment by jjangga0214** [comment]
@darold May I ping?

**Comment by darold** [comment]
Hi, a new release is planned at end of the first week of September.

**Comment by darold** [comment]
Sorry for the delay, I just published release v4.1

**Comment by bradymholt** [comment]
Thanks @darold - I just published [vscode-pgFormatter](https://marketplace.visualstudio.com/items?itemName=bradymholt.pgFormatter) version 1.8.0 that includes pgFormatter 4.1.
